Dockerfile

```diff
@@ -29,7 +29,7 @@ ENV PROJECT_HOME /usr/src
 ENV PROJECT_NAME universal-application-tool-0.0.1
 
 COPY ${PROJECT_NAME} ${PROJECT_HOME}/${PROJECT_NAME}
-RUN cd $PROJECT_HOME/$PROJECT_NAME && sbt reload
+RUN cd $PROJECT_HOME/$PROJECT_NAME && sbt reload compile
```

Comment:
I don't know why we removed `compile` when we set up tailwind. Let me know if compiling here breaks anything.

Reply:
I don't actually know why either... But if you tested this locally, then I guess it's probably fine?